app/src/main/res/values-b+zh+CN/localized.xml

```diff
@@ -288,6 +288,8 @@
 
     <string name="times_rolled">已抽取 %d 次</string>
 
+    <string name="ces_dropped">礼装掉落: %d</string>
```

Comment:
Are you sure that this is correct? If not, leave it to the Chinese devs who occasionally translate new features.

Reply:
I'll roll this one back, sorry android studio was giving me an error saying it wasnt translated to chinese.